src/model.py

```diff
@@ -71,6 +71,26 @@ def prettyprint(self) -> str:
         return f"RsaKey n={self.n} e={self.e}{d_part}"
 
 
+class EcdsaCurve:
+    """ Represents a ECDSA curve used by malware"""
+
+    def __init__(self, t: str, x: str, y: str) -> None:
+        self.t = t
+        self.x = x
+        self.y = y
+
+    def to_misp(self) -> MISPObject:
+        co = MISPObject("crypto-material", standalone=False)
+        co.add_attribute("type", "ECDSA")
+        co.add_attribute("ecdsa-type", self.t)
```

Comment:
I'm a bit torn on this, because technically `ecdsa-type` is an enum, and `ecdsa_pub_p384` (the value our configs have) is not a member of it.

Maybe consider special-casing it (see suggestion) 

```suggestion
        if self.t == "ecdsa_pub_p384":
            co.add_attribute("ecdsa-type", "NIST P-384")
        else:
            co.add_attribute("ecdsa-type", self.t)
```